src/nft/components/bag/bag.test.tsx

```diff
@@ -15,7 +16,11 @@ jest.mock('@web3-react/core', () => {
 
 describe('Bag.tsx', () => {
   it('matches base snapshot', () => {
-    const { asFragment } = render(<Bag />)
+    const { asFragment } = render(
+      <MockedProvider>
+        <Bag />
+      </MockedProvider>
+    )
```

Comment:
I think you actually want to add this to `WithProviders` in the `test-utils` file